upload-api/tables/index.js

```diff
@@ -26,3 +26,65 @@ export const uploadTableProps = {
   // space + root must be unique to satisfy index constraint
   primaryIndex: { partitionKey: 'space', sortKey: 'root' },
 }
+
+/** @type TableProps */
+export const delegationTableProps = {
+  fields: {
+    cid: 'string',        // `baf...x`
```

Comment:
we call cid link in other tables. Should we do the same here?

Reply:
works for me!